src/histogram-date-range.ts

```diff
@@ -122,6 +124,17 @@ export class HistogramDateRange extends LitElement {
     this.requestUpdate();
   }
 
+  private get hasValidData(): boolean {
+    return Boolean(
+      this.data.minDate &&
+        dayjs(this.data.minDate).valueOf() &&
+        this.data.maxDate &&
+        dayjs(this.data.maxDate).valueOf() &&
+        Array.isArray(this.data.bins) &&
```

Comment:
What other type could `this.data.bins` be?

Reply:
if someone passed in a malformed data object, eg `data.bins = "xyz"` I wanted it to no blow up when I try to `map` that property later on